cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -75,7 +75,29 @@ void SensorBridge::HandleOdometryMessage(
 
 void SensorBridge::HandleNavSatFixMessage(
     const std::string& sensor_id, const sensor_msgs::NavSatFix::ConstPtr& msg) {
-  LOG(FATAL) << "TODO(spielawa / wohe): NavSatFix support not yet implemented.";
+  const carto::common::Time time = FromRos(msg->header.stamp);
```

Comment:
Consistency: Change all uses of `cartographer::` and `::cartographer::` to `carto::`?